dask/utils.py

```diff
@@ -223,9 +227,11 @@ def changed_cwd(new_cwd):
 
 @contextmanager
 def tmp_cwd(dir=None):
-    with tmpdir(dir) as dirname:
-        with changed_cwd(dirname):
-            yield dirname
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=FutureWarning)
+        with tmpdir(dir) as dirname:
```

Comment:
Similar comment here about not using `tmpdir`

Reply:
My reply on the other comment also applies here